### PR TITLE
Hackathon add sponsor logos

### LINKED ISSUE #4075: Build - Add Logos to Hackathon Landing Page
### User Story

As a hackathon participant I want see logos of sponsors/funders.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: Hackers want to see who is sponsoring the prizes. Sponsors want to be sure that their logos can be found if people look for them.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
Display logos in two sizes on the hackathon bounties list page

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

- Build in two rows of logos on the bounties list page (one row of small, one row of larger)
- Upload logos in admin is okay, engineering to advise on simplest solution

### Data Requirements
TBD

Review with @PixelantDesign @alexvotofuture

### PR DESCRIPTION
**Pull request by octavioamu**
##### Checklist

- [ ] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [ ] Linter status: 100% pass
- [ ] Added relevant tests
- [ ] Tests pass and code coverage has not decreased
- [ ] Changes don't break existing behavior
- [ ] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [ ] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

##### Refers/Fixes

Fixes #4075
###### Contributor

- [ ] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Tested all changes **locally**
- [ ] Verified existing functionality
- [ ] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging or local deployment

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

**Comment by thelostone-mc** [comment]
@octavioamu could we compress the svg ?

**Comment by octavioamu** [comment]
@thelostone-mc ready to merge!!! Can you merge it so we deploy it tomo?

File: app/dashboard/templates/dashboard/hackathon/ethhack_2019.html [hunk 1]
```diff
@@ -26,6 +26,10 @@ <h1 class="font-weight-bolder my-5">Ethereal Virtual Hackathon</h1>
           </div>
         </div>
       </div>
+      <div class="row">
+          {% include 'shared/hackathon_sponsors.html' %}
+
```
Comment (danlipert) [inline app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: Extra space here

File: app/dashboard/templates/dashboard/index.html [hunk 2]
```diff
@@ -37,7 +37,7 @@
     {% if hackathon %}
       <div class="row pt-5 pb-5 text-center" style="
         background: rgb(25, 0, 63);
-        background-image: url({% static '/v2/images/header-bg.png' %});
+        background-image: url({% static 'v2/images/header-bg.png' %});
```
Comment (danlipert) [inline app/dashboard/templates/dashboard/index.html]: whats with this change?

File: app/dashboard/templates/shared/hackathon_sponsors.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<style>
+  .img-cover {
+    /* object-fit: cover; */
```
Comment (danlipert) [inline app/dashboard/templates/shared/hackathon_sponsors.html]: leftover comment

File: app/dashboard/templates/shared/hackathon_sponsors.html [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    <div class="img-holder rounded-circle img-holder--light">
+      <img src="{% static '/v2/images/hackathon-sponsors/small/autom8.svg' %}" alt="autom(8)" title="autom(8)" class="mw-100 mh-100 img-cover" width="75" height="75">
+    </div>
+    <div class="img-holder rounded-circle img-holder--light">
+      <img src="{% static '/v2/images/hackathon-sponsors/small/logo_poa.svg' %}" alt="POA" title="POA" class="mw-100 mh-100 img-cover" width="75" height="">
+    </div>
+    <div class="img-holder rounded-circle img-holder--light">
+      <img src="{% static '/v2/images/hackathon-sponsors/small/fluence-h.svg' %}" alt="fluence" title="Fluence" class="mw-100 mh-100 img-cover" width="75" height="75">
+    </div>
+
```
Comment (danlipert) [inline app/dashboard/templates/shared/hackathon_sponsors.html]: extra line here

File: app/dashboard/templates/shared/hackathon_sponsors.html [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    width: 75px;
+    height: 75px;
+    padding: 0.8em;
+  }
+</style>
+<div class="col-12 sponsors text-white mt-3 mb-5">
+  Supported by
+  <div class="big-sponsors d-flex flex-wrap justify-content-center mt-4">
+    <div class="img-holder rounded-circle img-holder--dark">
+      <img src="{% static '/v2/images/hackathon-sponsors/big/quorum.svg' %}" alt="quorum" title="quorum" class="mw-100 mh-100 img-cover" width="113" height="113">
```
Comment (danlipert) [inline app/dashboard/templates/shared/hackathon_sponsors.html]: Why are some of the `alt` tags not capitalized (and this `title` tag)?

File: app/dashboard/templates/dashboard/index.html (hunk 2 above)
Comment (octavioamu) [reply app/dashboard/templates/dashboard/index.html]: because with slash fails in production

Comment (danlipert) [reply app/dashboard/templates/dashboard/index.html]: lol makes sense! :)

**Pull request merged**